### PR TITLE
Doc refresh

### PR DESCRIPTION
**Pull request by davidx**
Update doc text and image to reflect changes in behavior.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,95 +5,103 @@ A simplified integration robot.
...
+## What's this ?
+Thumbs was created to help validate and merge pull requests. It was initially created as a replacement for Bors.
+## Setup
+#### 1) Enable Thumbs Webhook for your repository
+
+##### Github: Enable Webhook
+- Go to https://github.com/basho/YOUR-REPO/settings
+- Click [Webhooks]
+- Click [Add Webhook]
+- Set Payload URL to: http://thumbs.cloud1.basho.com
```
Comment (ooshlablu) [inline README.md]: I think this needs to be `http//thumbs.cloud1.basho.com/webhook`

Comment (davidx) [reply README.md]: Oh yeh for sure.

**Comment by paegun** [comment]
+1

**Comment by ooshlablu** [comment]
:+1:

**Pull request merged**